_data/devdocs/en/guides/contracts.md

```diff
@@ -210,20 +210,20 @@ all under an Apache license.
 
 {% autocrossref %}
 
-Alice is concerned about her privacy.  She knows every transaction gets
-added to the public block chain, so when Bob and Charlie pay her, they
+Alice is concerned about her privacy. She knows every transaction gets
+added to the public Blockchain, so when Bob and Charlie pay her, they
 can each easily track those satoshis to learn what Bitcoin
 addresses she pays, how much she pays them, and possibly how many
 satoshis she has left.
 
-Alice isn't a criminal, she just wants plausible deniability about
+Alice is not a criminal, she just wants plausible deniability about
 where she has spent her satoshis and how many she has left, so she
-starts up the Tor anonymity service on her computer and logs into an
+starts up the Tor anonymity network on her computer and logs into an
```

Comment:
The previous phrasing was correct.  Tor is an anonymity network, but the program you start on your computer is a service.